src/lib/core/a11y/list-key-manager.spec.ts

```diff
@@ -24,43 +26,32 @@ class FakeQueryList<T> extends QueryList<T> {
   }
 }
 
-export class FakeEvent {
-  defaultPrevented: boolean = false;
-  constructor(public keyCode: number) {}
-  preventDefault() {
-    this.defaultPrevented = true;
-  }
-}
 
 describe('Key managers', () => {
   let itemList: FakeQueryList<any>;
-  let DOWN_ARROW_EVENT: KeyboardEvent;
-  let UP_ARROW_EVENT: KeyboardEvent;
-  let TAB_EVENT: KeyboardEvent;
-  let HOME_EVENT: KeyboardEvent;
-  let END_EVENT: KeyboardEvent;
+  let EVENTS: {
```

Comment:
How about just `fakeKeyEvents`? It's not constant since you write the value in `beforeEach`.